google/gax/auth.py

```diff
@@ -31,19 +31,49 @@
 
 from __future__ import absolute_import
 
+import grpc
```

Comment:
That makes sense. Let's just get rid of the `auth` module altogether. If I recall correctly, we only created it to begin with because `oauth2client` didn't provide the generic auth function; since `google-auth` is going to do that, we shouldn't expose the functionality on the `gax` surface.